webapp/src/components/sidebar_right/gitlab_items.tsx

```diff
@@ -0,0 +1,278 @@
+import React from 'react';
+import {GitPullRequestIcon, IssueOpenedIcon, IconProps} from '@primer/octicons-react';
+import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
+import {Badge, Tooltip, OverlayTrigger} from 'react-bootstrap';
+import * as CSS from 'csstype';
+
+import CrossIcon from 'src/images/icons/cross';
+import DotIcon from 'src/images/icons/dot';
+import TickIcon from 'src/images/icons/tick';
+import SignIcon from 'src/images/icons/sign';
+import {formatTimeSince} from 'src/utils/date_utils';
+import {GitlabItemsProps, Label} from 'src/types/gitlab_items';
+
+export const notificationReasons: Record<string, string> = {
+    assigned: 'You were assigned to the issue/merge request',
+    review_requested: 'You were requested to review a merge request.',
+    mentioned: 'You were specifically @mentioned in the content.',
+    build_failed: 'GitLab build was failed.',
+    marked: 'Task is marked as done.',
+    approval_required: 'Your approval is required on this issue/merge request.',
+    unmergeable: 'This merge request can not be merged.',
+    directly_addressed: 'You were directly addressed.',
```

Comment:
This string's meaning is unclear. Under what circumstances would a user see this message?

Reply:
removing this notification reason as it is already removed
reference: https://gitlab.com/gitlab-org/gitlab/-/merge_requests/86109